tests/sles4sap/trento/test_hana_unregister_node.pm

```diff
@@ -0,0 +1,87 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Trento unregister the secondary node
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use base 'consoletest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils 'script_retry';
```

Comment:
seems this file is not using `script_retry`

Reply:
thx for the comment, will remove script_retry